### PR TITLE
Revert "bump o/k for the fixed scheduling tests"

### PR DESCRIPTION
**Pull request by dgoodwin**
[TRT-430](https://issues.redhat.com//browse/TRT-430)

Reverts openshift/origin#27323

Appears to be an issue with the use of k8s.gcr.io/e2e-test-images/agnhost:2.36 after the o/k bump in source PR. 4.12 nightly payloads are now blocked.

To re-introduce, you likely will want to unrevert this commit, layer in #27329, and then make sure ci/prow/e2e-metal-ipi-ovn-ipv6  passes, this failed on source PR and exposed the problem but it is an optional job and was ignored.

### CODE REVIEW

**Comment by stbenjam** [comment]
/lgtm

Let’s get this through ci. We can abort if someone mirrors before ci completes.

**Comment by stlaz** [comment]
/hold
Such a revert should explicitly mention which tests failed and should be run on a re-revert.

**Comment by dgoodwin** [comment]
Details in https://issues.redhat.com/browse/TRT-430

**Comment by dgoodwin** [comment]
Test to re-run is just to see ci/prow/e2e-metal-ipi-ovn-ipv6  pass, it failed on original PR and exposed the problem, but is an optional job and the failure ignored.

**Comment by dgoodwin** [comment]
Discussed with the team, https://github.com/openshift/origin/pull/27329 probably fixes the issue, but we believe standard policy should apply here, we need to get payloads flowing again and don't want to run the risk of waiting when other problems might be lurking. Original commit updated with details.

/hold cancel
/override ci/prow/e2e-aws-single-node
/override ci/prow/e2e-gcp
/override ci/prow/e2e-metal-ipi-ovn-ipv6 
/override ci/prow/e2e-aws-serial 
/override ci/prow/e2e-aws-single-node-upgrade

**Comment by yselkowitz** [comment]
e2e-metal-ipi-ovn-ipv6 passed in #27329, so adding that to the unrevert should suffice.

File: test/extended/util/image/image.go [hunk 1]
```diff
@@ -53,7 +53,7 @@ func init() {
 
 		// allowed upstream kube images - index and value must match upstream or
 		// tests will fail (vendor/k8s.io/kubernetes/test/utils/image/manifest.go)
-		"k8s.gcr.io/e2e-test-images/agnhost:2.39": 1,
+		"k8s.gcr.io/e2e-test-images/agnhost:2.36": 1,
```
Comment (yselkowitz) [inline test/extended/util/image/image.go:56]: Also for the unrevert, please consider adding a comment here to remind people to simultaneously update all other `agnhost` references in `test/extended/testdata/` (which then requires a `make update`) when updating this line for kube rebases.

**Pull request merged**